ethereum/contracts/Implementation.sol

```diff
@@ -32,6 +32,31 @@ contract Implementation is Governance {
 
     function initialize() initializer public virtual {
         // this function needs to be exposed for an upgrade to pass
+        uint256 evmChainId;
+        uint16 chain = _state.provider.chainId;
+
+        // Wormhole chain ids explicitly enumerated
```

Comment:
Should we actually put this into the core bridge or just application layer?

Reply:
In case we want to put it into the core bridge, we could also think about a getter function isForked() on the core bridge that the application layer could refer to
cc @hendrikhofstadt

Reply:
The core bridge could refuse attestations on a forked chain, but I think it should allow them for governance VAAs in case we want to upgrade to support the forked chain.